src/MissingPageRouter.php

```diff
@@ -33,6 +33,8 @@ public function getRedirectFor(Request $request)
 
         collect($redirects)->each(function ($redirects, $missingUrl) {
             $this->router->get($missingUrl, function () use ($redirects) {
```

Comment:
Found the probable root of the problem that is caused the tests to fail. The closure doesn't have access to `$missingUrl`. Add that variable to `use`.